Configuration/Settings.yaml

```diff
@@ -2,3 +2,7 @@ Neos:
   Fusion:
     defaultContext:
       Form: 'Neos\Fusion\Form\Eel\FormHelper'
+
+      # NOT public API
+      'Neos.Fusion.Form.Internal.ObjectAccess': 'Neos\Fusion\Form\Eel\ObjectAccessHelper'
+      'Neos.Fusion.Form.Internal.IdentityExtraction': 'Neos\Fusion\Form\Eel\IdentityExtractionHelper'
```

Comment:
I am pretty sure we do not want to add more deeply nested eel helpers. They collide so easy with others. Why not use names like `__FormObjectAcess`.